test/functional/helpers/report-helper.js

```diff
@@ -0,0 +1,56 @@
+function isBrowserRelevant (test, browserAlias) {

```

Comment:
We don't need additional dir for this file. BTW, since it's exports just one function - `getTestError` this module should be named `get-test-error.js`.